api/server/server.go

```diff
@@ -790,30 +790,23 @@ func postContainersCreate(eng *engine.Engine, version version.Version, w http.Re
 		return err
 	}
 	var (
-		job          = eng.Job("create", r.Form.Get("name"))
-		outWarnings  []string
-		stdoutBuffer = bytes.NewBuffer(nil)
-		warnings     = bytes.NewBuffer(nil)
+		warnings []string
+		name     = r.Form.Get("name")
+		env      = new(engine.Env)
 	)
 
-	if err := job.DecodeEnv(r.Body); err != nil {
+	if err := env.Decode(r.Body); err != nil {
```

Comment:
I understand that the purpose of all this is to get rid of everything under /engine. I feel like it's better to do it in phases, that's what I though was the idea/plan:

1. Get rid of all references to Job.
2. Replace env and dynamic lookups with type safe decoding.

I rather follow that script than this other one:

1. Get rid of some job references.
2. Replace env and dynamic lookups with type safe decoding.
3. Replace env references in the remaining jobs with the new type safe decoding.
4. Get rid of remaining jobs references.